superduperdb/backends/ibis/query.py

```diff
@@ -156,10 +157,14 @@ def _get_all_fields(self, db):
         for tab in component_tables:
             fields_copy = tab.schema.fields.copy()
             if '_outputs' in tab.identifier and self.renamings:
-                model = tab.identifier.split('/')[1]
+                match = re.search(r"_outputs_(.*?)_(\d+)", tab.identifier)
+                if match:
+                    model = match.group(1)
+                else:
+                    raise Exception(f'Match model name error {tab.identifier}')
                 for k in self.renamings.values():
                     if (
-                        re.match(f'^_outputs/{model}/[0-9]+$', tab.identifier)
+                        re.match(f'^_outputs_{model}_[0-9]+$', tab.identifier)
```

Comment:
This should use the function above right?

Reply:
Done